### PR TITLE
Make sure you execute rake rdoc from rails root directory [ci skip]

### PR DESCRIPTION
**Pull request by axilleas**
\cc @senny

### CODE REVIEW

**Comment by senny** [comment]
@axilleas can you please combine this PR with #19933 and #19934 and squash the commits? We try to keep related changes together.

**Comment by axilleas** [comment]
Ah ok sure.

**Comment by senny** [comment]
@axilleas thanks man :yellow_heart:

**Comment by axilleas** [comment]
No problem, it's the commit messages that give me headaches so I try to split the PRs but I understand you :) It should be fixed.

**Comment by senny** [comment]
@axilleas feel free to use multiline-commit messages.

```
update instructions to build guides locally [ci skip]

  1.
  2.
  3. 
```

**Comment by fxn** [comment]
Good! Question, are you "into" a directory?

**Comment by axilleas** [comment]
@senny amended commit to match your recomendation.

@fxn actually I kinda copy-pasted from https://github.com/rails/rails/blob/master/guides/source/ruby_on_rails_guides_guidelines.md#generation :smile:

**Comment by fxn** [comment]
@axilleas yeah, you can "cd into", but "be into"... not sure about that one :).

File: guides/source/api_documentation_guidelines.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ RDoc
 ----
 
 The [Rails API documentation](http://api.rubyonrails.org) is generated with
-[RDoc](http://docs.seattlerb.org/rdoc/).
+[RDoc](http://docs.seattlerb.org/rdoc/). To generate it, make sure you are
+into the `rails` root directory, run `bundle install` and execute:
```
Comment (senny) [inline guides/source/api_documentation_guidelines.md]: I'm with @fxn. `into` feels off here.

> make sure you are in the rails root directory.

**Comment by axilleas** [comment]
Ah I see what you mean, updated.

File: guides/source/api_documentation_guidelines.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ RDoc
 ----
 
 The [Rails API documentation](http://api.rubyonrails.org) is generated with
-[RDoc](http://docs.seattlerb.org/rdoc/).
+[RDoc](http://docs.seattlerb.org/rdoc/). To generate it, make sure you are
+in the `rails` root directory, run `bundle install` and execute:
```
Comment (senny) [inline guides/source/api_documentation_guidelines.md]: probably better not to quote rails.

File: guides/source/ruby_on_rails_guides_guidelines.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ The guides and the API should be coherent and consistent where appropriate. In p
 HTML Guides
 -----------
 
-Before generating the guides, make sure that you have the latest version of Bundler installed on your system. As of this writing, you must install Bundler 1.3.5 on your device.
+Before generating the guides, make sure that you have the latest version of Bundler installed on your system. As of this writing, you must install Bundler 1.3.5 or later on your device.
```
Comment (senny) [inline guides/source/ruby_on_rails_guides_guidelines.md]: Just out of curiosity, what didn't work when you used a older version of bundler?

Also could you hard-wrap that paragraph?

Comment (axilleas) [inline guides/source/ruby_on_rails_guides_guidelines.md]: It's not that it didn't work, but I think the way it is written now it kinda suggests that only v1.3.5 should be used.

File: guides/source/api_documentation_guidelines.md (hunk 2 above)
Comment (axilleas) [inline guides/source/api_documentation_guidelines.md]: Fair enough. I took it from the generation section of the HTML Guides where `guides` is also quoted ;)

File: guides/source/ruby_on_rails_guides_guidelines.md (hunk 3 above)
Comment (senny) [inline guides/source/ruby_on_rails_guides_guidelines.md]: @axilleas we only hard-wrap the parts, which have modifications anyway. This is to preserve the git-history. Meaning, we should wrap only that paragraph and in the same commit as the other changes.

File: guides/source/api_documentation_guidelines.md (hunk 2 above)
Comment (senny) [inline guides/source/api_documentation_guidelines.md]: yea but `guides/` is an actual dir, while `rails` just refers to the folder you cloned rails into. We can't really know what it's called.

**Comment by senny** [comment]
@axilleas thank you for your contribution.

**Comment by axilleas** [comment]
Thank you for your time and support :)

**Pull request merged**